source/_patterns/20-molecules/media-caption/media-caption.scss

```diff
@@ -10,6 +10,11 @@
   }
 }
 
+.media-caption--hover {
+  transition: background 200ms cubic-bezier(0.4, 0.0, 0.2, 1);
```

Comment:
Wäre es evtl. sinnvoll die Animationen in den variables zu speichern? brauchen wir so öfters.